flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-doris/src/main/java/org/apache/flink/cdc/connectors/doris/sink/DorisRowConverter.java

```diff
@@ -103,7 +103,8 @@ static SerializationConverter createExternalConverter(DataType type, ZoneId pipe
                 return (index, val) ->
                         val.getTimestamp(index, DataTypeChecks.getPrecision(type))
                                 .toLocalDateTime()
-                                .format(DorisEventSerializer.DATE_TIME_FORMATTER);
+                                .toString()
+                                .replace('T', ' ');
```

Comment:
What about fixing `DorisEventSerializer.DATE_TIME_FORMATTER` instead of doing string-level operations? It should make code clearer and more efficient.

Reply:
@yuxiqian ok